### ISSUE TITLE
Using multiple-files in a monorepo setup logs a missing summary error in annotations

### ISSUE THREAD

**Issue by tctc91** [MishaKav/jest-coverage-comment#66]
## Issue
In our monorepo setup, we use the `multiple-files` argument to reference multiple json summary reports whilst omitting the `coverage-summary-path` argument. E.g.

```
            - name: Jest coverage comment
              uses: MishaKav/jest-coverage-comment@main
              with:
                  title: Jest Unit Test Coverage
                  multiple-files: |
                      ui, packages/ui/coverage/coverage-summary.json
                      bff, packages/bff/coverage/coverage-summary.json

```

In our workflow annotations, we get the following log:

<img width="566" alt="Screenshot 2023-02-02 at 17 07 27" src="https://user-images.githubusercontent.com/456966/216394020-8b3d004d-33ce-496a-9151-f6300dd906f5.png">

## Solution
Suppress this log when `multiple-files` are included.

Happy to raise a PR for this :)

**Comment by MishaKav** [comment]
I believe the action works **as expected**, you just show a warning. Am I right?

**Comment by tctc91** [comment]
> I believe the action works **as expected**, you just show a warning. Am I right?

Yes exactly.

**Comment by MishaKav** [comment]
Thanks, I will fix it

**Comment by MishaKav** [comment]
Release a fix #68 for this, you can use:
 
```yaml
- name: Jest coverage comment
  uses: MishaKav/jest-coverage-comment@main
```
or 
```yaml
- name: Jest coverage comment
  uses: MishaKav/jest-coverage-comment@v1.0.23
```
